examples/device-dashboard/main.c

```diff
@@ -1,26 +1,31 @@
-// Copyright (c) 2020-2022 Cesanta Software Limited
+// Copyright (c) 2020-2023 Cesanta Software Limited
 // All rights reserved
 
-#include "mongoose.h"
+#include "net.h"
 
```

Comment:
I'd rather show including `mongoose.h` here, we are using Mongoose calls in this file and it is an example

Reply:
Makes sense, yes